src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -666,6 +669,30 @@ var DynamicTable = new Class({
                     }
                     return false;
                 });
+                tr.addEvent('touchstart', function(e) {
+                    console.log("event: touchstart");
```

Comment:
I don't think you really want this (and in other places).

Reply:
My mistake, removing this